stdlib/date/date.go

```diff
@@ -287,13 +287,26 @@ func init() {
 					return nil, errors.New(codes.Invalid, "missing argument unit")
 				}
 
-				if v.Type().Nature() == semantic.Time && u.Type().Nature() == semantic.Duration {
-					w, err := execute.NewWindow(u.Duration(), u.Duration(), execute.Duration{})
-					if err != nil {
-						return nil, err
+				if values.IsTimeable(v) && u.Type().Nature() == semantic.Duration {
```

Comment:
I think my understanding was that we're moving this section into a separate PR. Is that correct?

Reply:
You are right, my bad. I will just replace that file with the current version from master branch.